spec/models/ip_block_spec.rb

```diff
@@ -7,6 +7,13 @@
     it { is_expected.to validate_presence_of(:ip) }
     it { is_expected.to validate_presence_of(:severity) }
 
+    it 'validates severity inclusion', :aggregate_failures do
```

Comment:
Minor style thing - look at the `allow_values` validation spec helper used in some other spots -- could roll this up into basically a one-liner.

Reply:
Thanks for the suggestion. That's a lot cleaner. I refactored the test in e46423b4ac2063285873fb340771d49a57fa0ae7.